web/lib/auth.js

```diff
@@ -16,8 +17,12 @@ export function login({ email, password }) {
   return supabase.auth.signIn({ email, password })
 }
 
-export function logout() {
-  return supabase.auth.signOut()
+export async function logout() {
+  const { error } = await supabase.auth.signOut()
+
+  if (error) throw error
```

Comment:
Should handle it somehow (or silently ignore it at all) 🤔?

Reply:
mean no error?

Reply:
Yep. Let's assume our code is perfect.